inbox/config.py

```diff
@@ -85,7 +85,7 @@ def _update_config_from_env(config, env):
 
     if env in ['prod', 'staging']:
         base_cfg_path = [
-            '/etc/inboxapp/secrets.yml',
+            '/etc/inboxapp/secrets.json',
```

Comment:
I merged **config.json** and **secrets.yml** into single file: **config.json**

Also, I rename **secrets.yml** into **secrets.json** to keep config format consistent.

**secrets.json** is used for overriding any specific key from **config.json**.